tests/test_unit/test_compression/test_compress.py

```diff
@@ -0,0 +1,43 @@
+import os
+import tempfile
+
+import pytest
+import sleap_io as sio
+from movement import sample_data
+
+from ethology.io.compress import compress_video_h264
+
+
+@pytest.fixture
+def test_video():
+    """Fetches a sample dataset and returns video information."""
+    ds = sample_data.fetch_dataset(
+        "SLEAP_three-mice_Aeon_proofread.analysis.h5", with_video=True
+    )
+    video = sio.load_video(ds.video_path)
+    return video
+
+
+def test_compress_video_h264(test_video):
```

Comment:
Very nice that you added a test! However, we are not really testing if the video is compressed right? If `compress_video_h264` simply returned the same input video, the test would still pass.

Can you think of a way that would make us certain that the function is behaving as expected?